HeavyIonsAnalysis/JetAnalysis/src/HiInclusiveJetAnalyzer.cc

```diff
@@ -1464,6 +1481,25 @@ HiInclusiveJetAnalyzer::analyze(const Event& iEvent,
 	azimuth_adapt = azimuth_adapt_new;
       }
     }
+
+    //recluster the jet constituents in reWTA scheme-------------------------
+    // disable for calo jets
+    if(doWTARecluster_ && !(jetName_.find("Calo") != std::string::npos)){
+      std::vector<fastjet::PseudoJet> candidates;
+      auto daughters = jet.getJetConstituents();
+      for(auto it = daughters.begin(); it!=daughters.end(); ++it){
+        candidates.push_back(fastjet::PseudoJet((**it).px(), (**it).py(), (**it).pz(), (**it).energy()));
+      }
+      auto cs = new fastjet::ClusterSequence(candidates, WTAjtDef);
+      std::vector<fastjet::PseudoJet> wtajt = fastjet::sorted_by_pt(cs->inclusive_jets(0));
+
+      jets_.WTAeta[jets_.nref] = wtajt[0].eta();
+      jets_.WTAphi[jets_.nref] = wtajt[0].phi_std();
+      wtajt.clear();
```

Comment:
Why do you clear this vector?  It will go out of scope in two lines...

Reply:
I think you are right and this line can be safely removed.

Reply:
(There is another one down below that I didn't comment on)